### PR TITLE
:bug: :rocket: Doccano: Set ingress.path as a templated value

### PR DESCRIPTION
**Pull request by Axelcouty**
Proposal for https://github.com/curie-data-factory/helm-charts/issues/19

### CODE REVIEW

File: charts/doccano/templates/ingress.yaml [hunk 1]
```diff
@@ -21,11 +21,12 @@ spec:
 {{ toYaml .Values.ingress.tls | indent 4 }}
 {{- end }}
   rules:
+{{- $path := .Values.ingress.path -}}
```
Comment (Axelcouty) [inline charts/doccano/templates/ingress.yaml:24]: As we do a `{{- range .Values.ingress.hostnames }}` it seems that we cannot access .Values below line 25.

If you are good with this solution we can live it there, or we can also put this line on top of the file, what do you think ?

Comment (armandleopold) [reply charts/doccano/templates/ingress.yaml:24]: Yep that's right, you can only access values that are within the ranged object like in a for you have variable scoping, and you cannot access variables that are out of scope.
I prefer that you put it on top with the others.

Comment (armandleopold) [reply charts/doccano/templates/ingress.yaml:24]: You should also increase the Chart version from 1.3.6 to -> 1.3.7 in the `chart.yaml` thanks.

**Comment by armandleopold** [comment]
@Axelcouty  You should also increase the Chart version from 1.3.6 to -> 1.3.7 in the chart.yaml thanks.

**Pull request merged**